pychromecast/controllers/homeassistant.py

```diff
@@ -45,6 +47,7 @@ def hass_connected(self):
             self.status is not None
             and self.status["connected"]
             and self.status["hassUrl"] == self.hass_url
+            and self.status["hassUUID"] == self.hass_uuid
```

Comment:
Why is this necessary? In what case are the URLs the same but the UUIDs not?

Reply:
When developing or testing with multiple hass instances this can be the case.